x-pack/plugins/apm/scripts/bootstrap_apm/src/run.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+set -u
+scriptdir="$(dirname "$0")"
+cd "$scriptdir"
+source variables.sh
+source utils/logger.sh
+
+helpFunction()
+{
+   echo ""
+   echo "Usage: $0 -v "7.17.0" -n my-deployment -r europe-west2 -p gcp-hot-warm"
+   echo -e "\t-v Version of the stack "
+   echo -e "\t-n Deployment name"
+   echo -e "\t-r Region"
+   echo -e "\t-p Hardware profile"
+   exit 0 # Exit script after printing help
+}
+
+if ! [ -x "$(command -v ecctl)" ]; then
+  show_msg "[ERROR] ecctl not found. Download it here: https://www.elastic.co/downloads/ecctl " 4
+  exit 1
+fi
+
+if ! test -f "$ECCTL_CONFIG"; then
+   show_msg "[ERROR] Configuration file $ECCTL_CONFIG is missing." 4
+   exit 1
+fi
+
+while getopts ":v:n:r:p:" opt
+do
+   case "$opt" in
+      v ) VERSION="$OPTARG" ;;
+      n ) DEPLOYMENT_NAME="$OPTARG" ;;
+      r ) REGION="$OPTARG" ;;
+      p ) HARDWARE_PROFILE="$OPTARG" ;;
+      ? ) helpFunction ;; 
+   esac
+done
+
+show_msg "[INFO] Creating your deployment ${DEPLOYMENT_NAME}. Please hold, it takes about 5 minutes. ⌛" 
+
+# create deployment
+`ecctl deployment create -f ./config/deployment.json --version ${VERSION} --name ${DEPLOYMENT_NAME} \
```

Comment:
We should ensure `output.txt` is written to a temporary location (outside the git repository to prevent accidental commits) and always removed after the script is run through a `trap` https://www.linuxjournal.com/content/use-bash-trap-statement-cleanup-temporary-files

Reply:
fixed in https://github.com/elastic/kibana/pull/123791/commits/3696cfa7f07d8654013a63dd94f55cc62443e37b